frontend/src/main/scala/akkaviz/frontend/components/ActorStateTab.scala

```diff
@@ -2,10 +2,18 @@ package akkaviz.frontend.components
 
 import akkaviz.frontend.ActorPath
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2273483890/issues/source?bid=3121282&fileBranchId=3186015#l3)